packages/kbn-esql-ast/src/__tests__/ast_parser.inlinecast.test.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getAstAndSyntaxErrors as parse } from '../ast_parser';
+import { ESQLInlineCast, ESQLSingleAstItem } from '../types';
+
+describe('Inline cast (::)', () => {
+  describe('correctly formatted', () => {
+    it('can be a command argument', () => {
+      const text = 'FROM kibana_ecommerce_data | EVAL field::string';
+      const { ast, errors } = parse(text);
+
+      expect(errors.length).toBe(0);
+      expect(ast[1]).toMatchInlineSnapshot(`
```

Comment:
I am not a fan of inlineSnapshots. Which is the important things we want to test here? Why don't we test these properties instead here?

Reply:
Okay, makes sense. I will update this test!